docroot/modules/humsci/hs_layouts/hs_layouts.module

```diff
@@ -41,3 +43,15 @@ function hs_layouts_preprocess_container(&$variables) {
     $variables['#attached']['library'][] = 'hs_layouts/layout_builder_admin';
   }
 }
+
+/**
+ * Implements hook_preprocess_preprocess_pattern_spotlight().
+ */
+function hs_layouts_preprocess_pattern_spotlight(&$variables) {
+  $paragraph = $variables['context']->getProperty('entity');
```

Comment:
The getProperty method had to be located within the ui_patterns module which is contrib, but was being utilized by the PatternContext inside of the hs_layout modules use of patterns. This field is protected by default.